api/src/main/java/org/openmrs/module/fhir2/FhirConstants.java

```diff
@@ -313,4 +313,8 @@ private FhirConstants() {
 	public static final String EVERYTHING_SEARCH_HANDLER = "everything.search.handler";
 	
 	public static final String CONDITION_OBSERVATION_CONCEPT_UUID = "1284AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+	
+	public static final String SNOMED_SYSTEM_URI = "http://snomed.info/sct";
+	
+	public static final String RX_NORM_SYSTEM_URI = "http://www.nlm.nih.gov/research/umls/rxnorm";
```

Comment:
@ibacher can you confirm the above are correct and good to code in as constants?

Reply:
LGTM